### ISSUE TITLE
do we need both uuid and dc_identifier_s fields?

### ISSUE THREAD

**Issue by mejackreed** [geoblacklight/geoblacklight-schema#53]
These seem redundant to me, unless I'm missing something?

**Comment by andrewbattista** [comment]
I agree that these seem redundant. Mainly because I can't think of a scenario where we would want these two values to be different.

**Comment by drh-stanford** [comment]
`dc_identifer_s` is often the same as `uuid` but may be an alternate identifier. perhaps we don't need this flexibility since we already have `layer_id_s` and `layer_slug_s` which are alternate identifiers. in that case, i'd vote for dropping `uuid` in favor or `dc_identifier_s` to remain consistent with dublin core.

**Comment by andrewbattista** [comment]
I second that vote!

**Comment by mejackreed** [comment]
Shouldn't be too big a deal for GeoBlacklight https://github.com/geoblacklight/geoblacklight/search?utf8=%E2%9C%93&q=uuid
